QUEST_LV_0200.tsv

```diff
@@ -9436,12 +9436,12 @@ QUEST_LV_0200_20151001_009439	I will be okay if I take little more rest.{nl}I am
 QUEST_LV_0200_20151001_009440	$Stogas Commanding Officer Ades
 QUEST_LV_0200_20151001_009441	You've brought the letter from Mesafasla?{nl}Then what happened to Roches?{nl}
 QUEST_LV_0200_20151001_009442	Ah... It's fortunate to hear that you are okay.
-QUEST_LV_0200_20151001_009443	You are a foreigner for a long time {nl}The supplies has been stopping from the kingdom, we don't know what's going on..{nl}
+QUEST_LV_0200_20151001_009443	$You have been a foreigner for a long time {nl}There has been no supply from the kingdom, we don't know what's going on..{nl}
 QUEST_LV_0200_20151001_009444	Even I'm trying to stand by doing self-sufficiency, but It's hard to alive only myself.{nl}If you are okay, please solve the lack of foods.
 QUEST_LV_0200_20151001_009445	Please get the meat from Beadbird around here.{nl}Horas told me It has a good taste.{nl}
 QUEST_LV_0200_20151001_009446	We will starve, if the Beadbird.{nl}Here is too cold, so only Potatos are growing.
 QUEST_LV_0200_20151001_009447	Thank you.{nl}Firsly, we have to complete it.
-QUEST_LV_0200_20151001_009448	We are not only lacking food.{nl}The weapons also needed to be changed because of no supplies from camp.
+QUEST_LV_0200_20151001_009448	$We are not only lacking food.{nl}The weapons also needed to be improved because of no supplies from camp.
```

Comment:
Hmm, second sentence needs rewording.
"The weapons situation could also improve, as we've had no supplies from camp."